cmd/podman/run.go

```diff
@@ -83,8 +83,9 @@ func runCmd(c *cli.Context) error {
 	logrus.Debug("new container created ", ctr.ID())
 
 	if c.String("cidfile") != "" {
-		libpod.WriteFile(ctr.ID(), c.String("cidfile"))
-		return nil
+		if err := libpod.WriteFile(ctr.ID(), c.String("cidfile")); err != nil {
```

Comment:
Do we need this in podman create as well? I recall this code being there as well

Reply:
yup! fixed up too

Reply:
This is an error situation command should fial.  Return the error,

Reply:
the container is created at this point ... do you want me to delete the container then too?